src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/domain/Avslagskategori.kt

```diff
@@ -0,0 +1,73 @@
+package no.nav.tilleggsstonader.sak.vedtak.domain
+
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.ANNET
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.HAR_IKKE_MERUTGIFTER
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.HAR_IKKE_UTGIFTER
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.IKKE_I_MÅLGRUPPE
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.INGEN_AKTIVITET
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.INGEN_OVERLAPP_AKTIVITET_MÅLGRUPPE
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.MANGELFULL_DOKUMENTASJON
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.RETT_TIL_BOSTØTTE
+import no.nav.tilleggsstonader.sak.vedtak.domain.ÅrsakAvslag.RETT_TIL_UTSTYRSSTIPEND
+
+/**
+ * Kategorier som beskriver hvilket område en avslagsårsak tilhører
+ */
+enum class Avslagskategori {
+    AKTIVITET,
+    MÅLGRUPPE,
+    STØNADSVILKÅR,
+    GENERELL,
+}
+
+fun gyldigeAvslagsårsaker(
+    forStønadstype: Stønadstype,
+    basertPå: Avslagskategori,
+): Set<ÅrsakAvslag> =
+    when (forStønadstype) {
+        Stønadstype.BARNETILSYN ->
+            when (basertPå) {
+                Avslagskategori.AKTIVITET -> setOf(INGEN_AKTIVITET)
+                Avslagskategori.MÅLGRUPPE -> setOf(IKKE_I_MÅLGRUPPE)
+                Avslagskategori.STØNADSVILKÅR -> setOf(MANGELFULL_DOKUMENTASJON)
+                Avslagskategori.GENERELL -> setOf(ANNET)
```

Comment:
Burde alle ha `INGEN_OVERLAPP_AKTIVITET_MÅLGRUPPE` (?)

Reply:
Eller burde generell være en liste av "default", som inneholder
INGEN_AKTIVITET, IKKE_I_MÅLGRUPPE, INGEN_OVERLAPP_AKTIVITET_MÅLGRUPPE, ANNET

Og at 
`Avslagskategori.AKTIVITET -> `
inneholder stønadsspesifike årsaker til avslag for aktivitet?  

Alternativt
Avslagskategori.AKTIVITET -> årsakerAktivitet + setOf(<stønadsspesifike>)